docs/getting-started/docker-network-plugin/CalicoSwarm.md

```diff
@@ -39,45 +49,63 @@ Now, start the Calico node service on each node in the cluster.
+`<swarm_token>`. Your token should look something like this:
+
     d435dc104abdd89af24ae2392c7338ce
 
-## Configure Swarm Nodes
+## 5. Configure Swarm Nodes
 
-Before we configure our Swarm, we will need to start the Docker daemon and point the cluster-store to our running instance of etcd. 
+Before we configure our Swarm, we will need to start the Docker daemon and 
+point the cluster-store to our running instance of etcd. 
 
     $ sudo docker daemon -H tcp://0.0.0.0:2375 -H unix:///var/run/docker.sock --cluster-store=etcd://$MANAGER_IP:2379 &
 
-Now that we've got a token and a properly running daemon, we can begin to configure our Swarm. First, let's join each of our Swarm nodes to the cluster.
+Now that we've got a token and a properly running daemon, we can begin to 
+configure our Swarm. First, let's join each of our Swarm nodes to the cluster.
 
-Run the following commands on each node, replacing `<swarm_token>` with your given token.
+Run the following commands on each node, replacing `<swarm_token>` with your 
+given token.
     
     $ docker run -d swarm join --addr=$NODE_IP:2375 token://<swarm_token>
     
-## Configure Swarm Manager
+## 6. Configure Swarm Manager
 
-Let's now configure the Swarm manager. Note that `<swarm_port>` in the following command can be any unused TCP port on the manager server. This is the port the client will use to communicate with the Swarm manager daemon. 
+Let's now configure the Swarm manager. Note that `<swarm_port>` in the 
+following command can be any unused TCP port on the manager server. This is 
+the port the client will use to communicate with the Swarm manager daemon. 
 
     $ docker run -d -p <swarm_port>:2375 swarm manage token://<swarm_token>
     
-## Testing Docker Swarm
+## 7. Testing Docker Swarm
     
-At this point we should have a fully configured Calico networked Swarm cluster. However, there are no workloads running on our cluster. Let's create a few containers and check their connectivity. We can run the following commands on the client against the Swarm Manager using the -H flag.
+At this point we should have a fully configured Calico networked Swarm cluster.
+However, there are no workloads running on our cluster. Let's create a few 
+containers and check their connectivity. We can run the following commands on 
+the client against the Swarm Manager using the -H flag.
 
-First, set the SWARM_PORT environment variable on the client to the value chosen when configuring the Swarm manager.
+First, set the SWARM_PORT environment variable on the client to the value 
+chosen when configuring the Swarm manager.
 
     $ export SWARM_PORT=<swarm_port>
        
-Then, we will configure three networks using the Calico network and Calico IPAM drivers. Note that we need to first create a pool of allowable IP addresses for the containers. Here we create a pool with CIDR 192.168.0.0/16.
+Then, we will configure three networks using the Calico network and Calico 
```

Comment:
Shouldn't matter if it already exists - it'll just blat over the existing one.  I think this just simplifies teh text because you don't have to say oh look we already happen to have a pool with the correct CIDR, but makes it clear that having the pool is necessary.